colcon_core/package_augmentation/python.py

```diff
@@ -156,3 +162,22 @@ def _next_incompatible_version(version):
     if len(version) == 1:
         version.append(0)
     return '.'.join(map(str, version))
+
+
+def _extract_maintainers_with_emails(metadata):
+    if 'maintainer' in metadata:
+        maintainer = metadata['maintainer']
+        maintainer_email = metadata.get('maintainer_email')
+    else:
+        # If no explicit maintainer is given then it is likely that the
+        # original author is maintaining the package
```

Comment:
Assuming that the author is the maintainer is a bit optimistic. Not sure what is the end goal of the information but would be great if we could indicate somehow that this info is not as accurate as the one getting directly from  the `maintainer` tag.

Reply:
This behavior is based on recommended practices in the official Python packaging guide: https://packaging.python.org/en/latest/specifications/core-metadata/#maintainer